packages/database/src/migrations/20211128090124-MergeFlutrackingResponses-modifies-data.js

```diff
@@ -165,14 +157,24 @@ const mergeFlutrackingResponses = async (db, surveyCode) => {
     'Processing response groups in batches. Each batch will be processed as a transaction',
   );
   console.log(`Batches found: ${responseGroups.length}`);
+
+  let prevPercentage = -1;
   await processInBatches(responseGroups, RESPONSE_GROUP_BATCH_SIZE, async (responseGroup, i) => {
     await db.runSql('START TRANSACTION');
-    printProgress(i, responseGroups.length);
+
+    // Print progress
+    const percentage = getPercentage(i, responseGroups.length);
+    if (parseInt(percentage - prevPercentage) > 0) {
```

Comment:
The previous logic in `printProgress` allows us to update the last console output line with a changing percentage, which is cool, but for some reason doesn't work on the deployment logs.

What we now do here is printing an integer percentage every time the integer part changes (i.e. 100 times max per process)